application/shared-kernel/InfrastructureCore/InfrastructureCoreConfiguration.cs

```diff
@@ -11,9 +11,7 @@ public static class InfrastructureCoreConfiguration
 {
     private static string? _cachedConnectionString;
 
-    public static readonly bool SwaggerGenerator = File.Exists(
-        Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!, "SwaggerGenerator.tmp")
-    );
+    public static readonly bool SwaggerGenerator = Environment.GetEnvironmentVariable("SWAGGER_GENERATOR") == "true";
```

Comment:
Does this work on Windows? I was not able to configure environment variables just for one build. That why I've added the hack with the `SwaggerGenerator.tmp` temp file.

Reply:
It should, it's part of the MSBuild spec